subprojects/android-test/test-screenshot/src/main/java/com/avito/android/test/screenshot_test/internal/Screenshot.kt

```diff
@@ -68,8 +68,14 @@ internal class ScreenshotDirectory(
 
         private fun Context.getResolution(): String {
             val wm = getSystemService(Context.WINDOW_SERVICE) as WindowManager
+
+            // suggested method only available on 30+ API
+            @Suppress("DEPRECATION")
```

Comment:
Can we use `@RequiresApi` here to avoid false-negative?

Reply:
`@RequiresApi` not applicable to local variables, 
By adding it to `getResolution()` we will spread error about min api 30 all around.
What's the point?

Reply:
I see, I thought by mistake that this property has min API restriction.